backend/api/Pims.Api.csproj

```diff
@@ -47,6 +47,7 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
     <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="6.5.0" />
     <PackageReference Include="System.Drawing.Common" Version="7.0.0" />
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```

Comment:
Explicitly adding the package reference to the latest "RegularExpressions" version.